frontend/src/components/hub/HubBrowsePage.tsx

```diff
@@ -169,7 +169,7 @@ export default function HubBrowsePage({
   const [hubSupporters, setHubSupporters] = useState(null);
   const contentRef = useRef<HTMLDivElement>(null);
   // Do we need this? this line was removed on PR ##1560
-  const donationGoalActive = donationGoal && donationGoal.hub === hubUrl;
```

Comment:
This business logic is now owned by the DonationCampaignInformation component to reduce code duplication and complexity.